Makefile

```diff
@@ -270,6 +287,31 @@ bender-rm:
 	rm -f bender
 
 
+## --------------
+## RTL
+## --------------
+
+.PHONY: gen_rtl rtl_clean
+
+gen_rtl: $(RTL_CFGS)
+
+gen_rtl_%: util/idma_gen.py Makefile
```

Comment:
@TheMightyDuckOfDoom, is this still in line with the generator? Are all the switches covered here?

Reply:
Yes, only "--read-protocols", "--write-protocols" and "--shifter" are arguments